packages/custom-field/theme/material/vaadin-custom-field-styles.js

```diff
@@ -24,6 +24,11 @@ registerStyles(
       -moz-osx-font-smoothing: grayscale;
     }
 
+    /* TODO: remove when all components use new indicator */
```

Comment:
Can we have list of components that need the new indicator?

Reply:
Good point, updated TODO comments to mention the remaining components to be covered.